diff --git a/create-sdcard.sh b/create-sdcard.sh
index a1b1190..4f4e953 100755
--- a/create-sdcard.sh
+++ b/create-sdcard.sh
@@ -182,7 +182,7 @@ populate_3_partitions() {
         mkdir $PWD/tmp
 
         mount -t vfat ${DRIVE}${P}1 boot
-        mount -t ext3 ${DRIVE}${P}2 rootfs
+        mount         ${DRIVE}${P}2 rootfs
         mount -t ext3 ${DRIVE}${P}3 start_here
 
         # Remove any existing content in case the partitions were not
@@ -673,7 +673,7 @@ mkdir $PATH_TO_SDBOOT
 mkdir $PATH_TO_SDROOTFS
 
 sudo mount -t vfat ${DRIVE}${P}1 boot/
-sudo mount -t ext3 ${DRIVE}${P}2 rootfs/
+sudo mount         ${DRIVE}${P}2 rootfs/
 
 
 
@@ -690,8 +690,10 @@ sync
 sync
 sync
 
+build=*
 BOOTFILEPATH="$PARSEPATH/tisdk-debian-${build}-boot.tar.xz"
 ROOTFILEPATH="$PARSEPATH/tisdk-debian-${build}-rootfs.tar.xz"
+ROOTFSPATH="$PARSEPATH/tisdk-debian-${build}-rootfs/*"
 
 cat << EOM
 ################################################################################
@@ -710,7 +712,7 @@ echo ""
 sync
 
 echo "Copying rootfs System partition"
-untar_progress $ROOTFILEPATH $PATH_TO_SDROOTFS
+rsync -aHAX $ROOTFSPATH $PATH_TO_SDROOTFS
