--- scripts/build_distroiGOS.sh	2024-10-00 00:00:00.000000000 -0500
+++ scripts/build_distroiGOS.sh	2024-11-00 00:00:00.000000000 -0500
@@ -21,7 +21,7 @@
 
     cd ${topdir}/build/
 
-    ROOTFS_DIR=${topdir}/build/${distro}/tisdk-debian-${distro}}-rootfs
+    ROOTFS_DIR=${topdir}/build/${distro}/tisdk-debian-${distro}-rootfs
 }
 
 function package_and_clean() {
@@ -29,6 +29,8 @@
 bsp_version=$2
 
     cd ${topdir}/build/${build}
+    rm -rf tisdk-debian-${distro}-${bsp_version}-rootfs
+    cp -ra ../fs/ tisdk-debian-${distro}-${bsp_version}-rootfs
 
     log "> Cleaning up ${build}"
     tar --use-compress-program="pigz --best --recursive | pv" -cf tisdk-debian-${distro}-${bsp_version}-rootfs.tar.xz tisdk-debian-${distro}-${bsp_version}-rootfs &>>"${LOG_FILE}"
@@ -44,4 +46,3 @@
     log "> Packaging ${build}"
     tar --use-compress-program="pigz --best --recursive | pv" -cf ${distro}-${bsp_version}.tar.xz ${build} &>>"${LOG_FILE}"
 }
-
