diff --git a/build/linux/config/compiler.mk b/build/linux/config/compiler.mk
index e9e7f79..3868313 100644
--- a/build/linux/config/compiler.mk
+++ b/build/linux/config/compiler.mk
@@ -65,7 +65,7 @@ define calculate-compiler-preferred-target
    ifneq ($$(filter i386-% i486-% i586-% i686-%,$$($(1)_compiler_preferred_target)),)
     $(1)_compiler_preferred_target := i386-linux-gnu
    endif
-   ifneq ($$(filter aarch64-%-linux arm64-%,$$($(1)_compiler_preferred_target)),)
+   ifneq ($$(filter aarch64-% arm64-%,$$($(1)_compiler_preferred_target)),)
     $(1)_compiler_preferred_target := aarch64-linux-gnu
    endif
    ifneq ($$(filter armv7a-cros-linux-gnueabi armv7a-cros-linux-gnueabihf,$$($(1)_compiler_preferred_target)),)
